test/blackbox-tests/test-cases/dune

```diff
@@ -0,0 +1,86 @@
+(env
+ (_
+  (binaries ../utils/dune_cmd.exe)))
+
+(alias
+ (name test-deps)
+ (deps
+  ../utils/dune_cmd.exe
```

Comment:
Isn't the above line redundant with the bellow one?

Reply:
Good catch. Removed.